connect-examples/v2/csharp_checkout/csharp_checkout/Pages/Checkout.cshtml.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.Extensions.Configuration;
+using Square.Connect.Api;
+using Square.Connect.Model;
+
+namespace csharp_checkout.Pages
+{
+  public class CheckoutModel : PageModel
+  {
+    private readonly string locationId;
+
+    public CheckoutModel(IConfiguration configuration)
+    {
+      this.locationId = configuration["AppSettings:LocationId"];
+    }
+
+    public string ResultMessage
+    {
+      get;
+      set;
+    }
+
+    public IActionResult OnPost()
+    {
+      CheckoutApi checkoutApi = new CheckoutApi();
+      int amount = (int)float.Parse(Request.Form["amount"]) * 100;
+
+      try
+      {
+
+        CreateCheckoutRequest body = new CreateCheckoutRequest(
+          IdempotencyKey: Guid.NewGuid().ToString(),
+          Order: new CreateOrderRequest(
```

Comment:
Is this same as other checkout examples?

Can we call `CreateOrderRequest` to create a order before call `CreateCheckoutRequest `?